go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/containerd/containerd v1.0.2 // indirect
 	github.com/containerd/typeurl v0.0.0-20190228175220-2a93cfde8c20 // indirect
 	github.com/containernetworking/cni v0.6.0
-	github.com/coreos/etcd v3.3.10+incompatible
+	github.com/coreos/etcd v3.3.13-0.20190425194549-ad7c2cddb000+incompatible
```

Comment:
would prefer not to move off a tagged version for an untagged one. anything in flight in the 3.3.x stream we should be concerned about picking up half-baked?

Reply:
@liggitt how about i ask for a new release just to be sure :)